public/schema/JobTemplate.yaml

```diff
@@ -4,10 +4,10 @@ description: openQA job template
 type: object
 additionalProperties: false
 required:
-- architectures
 - products
+- test-suites
 properties:
-  architectures:
+  test-suites:
     description: A set of architectures with a list of test suite(s) eg. ppc64le
```

Comment:
Maybe `A list of test suites grouped by architectures` would be a better description?